R/utils.R

```diff
@@ -1,3 +1,20 @@
+recodeOverlayValues <- function(overlayData, desiredOverlayValues, variables) {
+  if (is.null(desiredOverlayValues)) return(overlayData)
+  dataType <- veupathUtils::findDataTypesFromPlotRef(variables, 'overlay')
+  
+  if (dataType %in% c('NUMBER', 'INTEGER', 'DATE')) {
+    # figure this out when i get to the continuous overlay PR, just leaving a skeleton for now
+    # tbh desiredOverlayValues will probably stop being a simple character vector then as well. but one thing at a time.
+    warning("Binned continuous overlays are not supported yet.")
+  } else {
+    if (unique(overlayData) == desiredOverlayValues) return(overlayData)
+    overlayData[!overlayData %in% desiredOverlayValues] <- 'All un-selected values'
```

Comment:
`Unselected` just one word I think?

I think the verbiage here might need some tweaking.  Imagine receiving a screenshot or share link where you don't know that the legend values were "selected".  Maybe return `__UNSELECTED__` and we figure it out client side?